build/profiles/freenas/ports-system.pyd

```diff
@@ -425,3 +425,39 @@ if DEBUG:
 	}
 	ports += "devel/ctags"
 	ports += "devel/py-pyflakes"
+
+if PRODUCT == "TrueNAS" or WITH_ASIGRA:
+    ports += {
+        "name": "databases/postgresql10-server",
+        "options": [
+            "OPTIONS_FILE_UNSET+=DEBUG",
+            "OPTIONS_FILE_UNSET+=DTRACE",
+            "OPTIONS_FILE_SET+=GSSAPI",
+            "OPTIONS_FILE_SET+=INTDATE",
+            "OPTIONS_FILE_SET+=LDAP",
+            "OPTIONS_FILE_SET+=NLS",
+            "OPTIONS_FILE_UNSET+=OPTIMIZED_CFLAGS",
+            "OPTIONS_FILE_UNSET+=PAM",
+            "OPTIONS_FILE_SET+=SSL",
+            "OPTIONS_FILE_SET+=TZDATA",
+            "OPTIONS_FILE_SET+=XML"
+        ]
+    }
+    ports += "databases/postgresql10-client"
+    ports += "databases/postgresql10-contrib"
+    ports += "databases/postgresql10-plpython"
+    ports += {
+        "name": "databases/py-psycopg2",
+        "options": [
+            "PGSQL_DEFAULT=10"
+        ]
+    }
+    ports += "devel/gettext-runtime"
+    ports += "devel/icu"
+    ports += "print/indexinfo"
+    ports += "textproc/libxml2"
+    ports += "lang/perl5.24"
```

Comment:
Dependencies should not be listed, its gathered automatically in poudriere build.

Reply:
How so? I understand how poudriere works, but what port is specifying these as dependencies? None that I am aware of. These are required by Asigra. I see no harm in explicitly stating them. If these aren't stated specifically, what is pulling them in?

Reply:
icu is a dependency if postgresql.
gettext-runtime is a dependency of dozens of packages.
indexinfo is a dependency of gettext and readline, among others.
libxml2 is a dependency of libxslt, lxml, apache, among others
perl is a dependency of glib, netatalk, mosh, among others